crates/chain/src/tx_graph.rs

```diff
@@ -971,10 +981,10 @@ impl<A: Anchor> TxGraph<A> {
     /// If the [`ChainOracle`] implementation (`chain`) fails, an error will be returned with the
     /// returned item.
     ///
-    /// If the [`ChainOracle`] is infallible, [`list_chain_txs`] can be used instead.
+    /// If the [`ChainOracle`] is infallible, [`list_canonical_transactions`] can be used instead.
     ///
-    /// [`list_chain_txs`]: Self::list_chain_txs
-    pub fn try_list_chain_txs<'a, C: ChainOracle + 'a>(
+    /// [`list_canonical_transactions`]: Self::list_canonical_transactions
+    pub fn try_list_canonical_transactions<'a, C: ChainOracle + 'a>(
```

Comment:
Many of our methods use `tx` as a name for `transaction`. What do you think about renaming this to `try_list_canonical_txs` for consistency?